README.md

```diff
@@ -2,25 +2,64 @@
 
 # operatorkit
 
-The operatorkit package implements an opinionated framework for developing
-Kubernetes operators. It can be used as a library in golang. It is built on
-top of our [microkit](https://github.com/giantswarm/microkit) framework which
-provides base functionality like logging and error handling.
+The operatorkit is a library for creating [Kubernetes operators][operators]. It
+emerged as extracted common functionality of number of the operators we
+developed in Giant Swarm. The goal of the library is to provide common
+structure of the operator projects and encapsulate best practices learned
+during running operators in production.
+
+## Features
```

Comment:
I would maybe differentiate between design goals and features in terms of packages operatorkit provides. We can link to packages. We cannot link to our good intentions though. Having these blocks separated would make sense to me logically and visually.

Reply:
I did it in `Current Scope`. Do you think it's redundant?
Maybe I rename `Current Scope` to `Features`. `Future Scope` to `Future Goals` and remove `Features`.
WDYT?

Reply:
Uncertain. Lets keep it as it is and adopt along the way.